src/cmd-buildextend-azure

```diff
@@ -150,7 +143,7 @@ def cli():
     # Identify the builds
     build = Build(
         os.path.join('builds'),
-        args.build,
+        args.build or 'latest',
```

Comment:
Hmm, or let's just standardize on `args.build` defaulting to `latest`? We do this in a couple of spots already.

Reply:
talked with @jlebon in IRC. Basically the azure file is the only one using the cosalib/build.py library. We could easily default to `latest` for args.build there but defaulting in the other files would be problematic because they aren't using cosalib/build.py yet. Ideally we'd move to that new library for the other commands but we're not doing that yet.